pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -494,10 +494,13 @@ func (r *ReconcileCincinnati) ensurePolicyEngineRoute(ctx context.Context, reqLo
 	}
 
 	// found existing resource; let's compare and update if needed
-	if !reflect.DeepEqual(found.Spec, route.Spec) {
+	if !reflect.DeepEqual(found.Spec.Port, route.Spec.Port) || !reflect.DeepEqual(found.Spec.To, route.Spec.To) {
 		reqLogger.Info("Updating Route", "Namespace", route.Namespace, "Name", route.Name)
 		updated := found.DeepCopy()
-		updated.Spec = route.Spec
+		// Update everything but updated.Spec.TLS. We want to allow user to update the TLS cert/key manually on the route
```

Comment:
Just sanity-checking; `TLS` is not the only field being ignored, right? Maybe it's worth a quick doc statement on why these are the only two fields we want to manage after creation?

Reply:
Port, To and TLS are the only things we configure. See. https://github.com/openshift/cincinnati-operator/blob/master/pkg/controller/cincinnati/new.go#L212-L222

Here's the output from the Route object. It adds the default `host` field and `wildcardPolicy` of None.
```
spec:
  host: example-cincinnati-policy-engine-route-openshift-cincinnati.apps-crc.testing
  port:
    targetPort: policy-engine
  tls:
    insecureEdgeTerminationPolicy: Allow
    termination: edge
  to:
    kind: Service
    name: example-cincinnati-policy-engine
    weight: 100
  wildcardPolicy: None
```

Reply:
Gotcha. So here by "everything" you mean "everything this operator originally set", not "everything in the Spec". I think it would be marginally more clear to make that distinction explicitly, but it's up to you.